src/style.css

```diff
@@ -0,0 +1,146 @@
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.container {
+  height: 406px;
+  width: 198px;
+    
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.photo {
+  display: flex;
+  height: 134px;
+  width: 160px;
+  padding-top: 33px;
+  margin: 0 auto 40px;
```

Comment:
You should set correct paddings to the card and use margin-top or margin-bottom for all elements